elements/pl-variable-description/pl-variable-description.py

```diff
@@ -0,0 +1,34 @@
+import prairielearn as pl
+import lxml.html
+import chevron
+
+
+VARIABLES_CATEGORY_DEFAULT = None
+
+
+def prepare(element_html, element_index, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    pl.check_attribs(element, ['variables-category'], [])
```

Comment:
We seem to be using `name` to refer to the keys in `data["params"]` and similar dictionaries. For example, `answers-name` is the attribute that other elements use to refer to the key in `data["correct_answers"]`. So maybe `variables-name` would be a good choice here?